app/controllers/katello/api/v2/content_view_filter_bulk_rules_controller.rb

```diff
@@ -0,0 +1,63 @@
+module Katello
+  class Api::V2::ContentViewFilterBulkRulesController < Api::V2::ApiController
+    before_action :find_filter
+    before_action :find_rule, :except => [:create]
+
+    api :POST, "/content_view_filters/:content_view_filter_id/bulk",
+        N_("Create a bunch of filter rules. The parameters included should be based upon the filter type.")
+    param :content_view_filter_id, :number, :desc => N_("filter identifier"), :required => true
+    param :data, Array, :of => [Hash], :desc => N_("hash for RPM bulkloads: \
+                                                    [{'name': 'name', \
+                                                    'version': 'version',\
+                                                    'architecture': 'architecture'}]") do
+      param :name, String, :desc => N_("package: name"), :required => true
```

Comment:
the `package: name` syntax makes me think I have to submit this as `package: foo`, can we just say `package name`?

Reply:
Just tried to follow the same style found at existing entry points... I'm changing this...

Reply:
fixed